app/src/main/java/com/dscvit/handly/ui/PostAuthActivity.kt

```diff
@@ -0,0 +1,57 @@
+package com.dscvit.handly.ui
+
+import android.os.Bundle
+import android.util.Log
+import androidx.appcompat.app.AppCompatActivity
+import androidx.navigation.NavOptions
+import androidx.navigation.Navigation
+import com.dscvit.handly.R
+import com.dscvit.handly.util.Constants
+import com.dscvit.handly.util.PrefHelper
+import com.google.android.material.tabs.TabLayout
+import com.onesignal.OneSignal
+import kotlinx.android.synthetic.main.activity_post_auth.*
+
+class PostAuthActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_post_auth)
+
+        val sharedPref = PrefHelper.customPrefs(this, Constants.PREF_NAME)
+        Log.d("esh", sharedPref.getString(Constants.PREF_AUTH_TOKEN, "")!!)
+        Log.d("esh", OneSignal.getPermissionSubscriptionState().subscriptionStatus.userId)
+
+        val navController = Navigation.findNavController(this, R.id.post_auth_nav_host)
+        val navOptions = NavOptions.Builder()
```

Comment:
You can set these nav options in the navigation graph I believe. Try using that instead

Reply:
Yeah, took this one from an example, will migrate to xml